common/connection.go

```diff
@@ -44,6 +44,26 @@ const wsWriteBufferSize = 1 << 20
 var _ EventEmitter = &Connection{}
 var _ cdp.Executor = &Connection{}
 
+type executorEmitter interface {
+	cdp.Executor
+	EventEmitter
+}
+
+type connection interface {
+	executorEmitter
+	Close(...goja.Value)
+	getSession(target.SessionID) *Session
+}
+
+type session interface {
+	cdp.Executor
+	executorEmitter
+	ExecuteWithoutExpectationOnReply(context.Context, string, easyjson.Marshaler, easyjson.Unmarshaler) error
+	SessionID() target.SessionID
```

Comment:
`session.SessionID()` stutters, so:

```suggestion
	ID() target.SessionID
```

Reply:
I did so because it's an interface and clash with other stuff in the future, but let's try.